src/Forms/Schema/FormSchema.php

```diff
@@ -112,9 +113,29 @@ public function getSchema(Form $form)
             $schema['fields'][] = $field->getSchemaData();
         }
 
+        // Validate there are react components for all fields
+        // Note 'actions' (FormActions) are always valid because FormAction.schemaComponent has a default value
+        $this->recursivelyValidateSchemaData($schema['fields']);
+
         return $schema;
     }
 
+    private function recursivelyValidateSchemaData(array $schemaData)
+    {
+        foreach ($schemaData as $data) {
+            if (!$data['schemaType'] && !$data['component']) {
+                $name = $data['name'];
+                $message = "Could not find a react component for field \"$name\"."
+                    . "Replace or remove the field instance from the field list,"
```

Comment:
```suggestion
                    . " Replace or remove the field instance from the field list,"
```
Need this space

Reply:
Should also be single quotes for this and the next line, but I won't hold the PR up if you forget to do that.

Reply:
Updated